### ISSUE TITLE
Hide Token/Balance red header in zero state & zero balance instances

### ISSUE THREAD

**Issue by jcisemer** [Dryec/tron-wallet-android#9]
Status quo: Token/Balance red header is displayed on wallet screen (1) in zero state when no 3rd party tokens have ever been added to a wallet and also (2) in zero balance instance after a 3rd party token balance has been drawn down to zero

Feature Request:
(1) in zero state without any 3rd party tokens added, do not display any Token/ Balance header as it confuses users without any tokens/balances below it
(2) in zero balance instance, if *all* 3rd party token balances are zero, do not display any Token/Balance header and zero balances to make UI cleaner and less cluttered
(3) if any 3rd party token balances greater than zero, continue to display Token/Balance header and balance amounts as is

In short, catch whether token balance is zero with an IF statement, and only display the red header for Token/Balance and the token list view below it when token balances are greater than zero.

**

P.S. Ansonsten: großartiger Job, Marius! Wirklich große Klasse, was Du hier gebaut hast!

![zero-state-zero-balance-large](https://user-images.githubusercontent.com/10479630/42741568-85383848-8881-11e8-915b-f236f60d94cb.png)

**Comment by jcisemer** [comment]
I am currently using Tron Wallet v1.1.4.4 where this issue is occurring.

**Comment by Dryec** [comment]
Thanks for that, I'll add these tomorrow!
